src/data.js

```diff
@@ -1,6 +1,94 @@
 import BigNumber from "bignumber.js";
 import * as Constants from "./constants";
 
+export function getContractFunctionIndex(contract_type, function_type) {
+    switch (contract_type) {
+        case 'TOKEN':
+            switch (function_type) {
+                case 'SUPERSEDE':
+                    return Constants.SUPERSEDE_FUNCIDX
+                case 'ISSUE':
+                    return Constants.ISSUE_FUNCIDX
+                case 'DESTROY':
+                    return Constants.DESTROY_FUNCIDX
+                case 'SPLIT':
+                    return Constants.SPLIT_FUNCIDX
+                case 'SEND':
+                    return Constants.SEND_FUNCIDX
+                case 'SEND_SPLIT':
```

Comment:
token separates to TOKEN and SPLITTABLE_TOKEN and no longer use SEND_SPLIT, just use SEND. If user want to send split support type token, he/she just call:

```
import * as ContractType from "./contract_type";
...
vsys.getContractFunctionIndex(ContractType.SPLITTABLE_TOKEN, 'SEND');
```